nextflow_schema.json

```diff
@@ -14,11 +14,10 @@
             "properties": {
                 "input": {
                     "type": "string",
-                    "format": "file-path",
+                    "format": "file-path-pattern",
```

Comment:
is this a new format?

Reply:
Allowing both csv and fasq with --input?
That seems not very nf-core-y BTW

Reply:
Yes, pointed out by @nvnieuwk :D 

It's not very modern nf-core-y indeed, but throw back from DSL1/very early DSL2 days (eager v2 does it too). I'm not removing it yet because that would be a backwards incompatible change requiring a major version bump etc. Will be removed in the future :)